src/components/v5/common/CompletedAction/partials/ManageTokens/partials/TokenStatusPill/TokenStatusPill.tsx

```diff
@@ -0,0 +1,31 @@
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+
+import { formatText } from '~utils/intl.ts';
+import PillsBase from '~v5/common/Pills/PillsBase.tsx';
+import { TokenStatus } from '~v5/common/types.ts';
+
+import { type TokenStatusPillProps } from './types.ts';
+
+const TokenStatusPill: FC<TokenStatusPillProps> = ({ status }) => {
+  const showNewLabel = status === TokenStatus.Added;
+  const showRemovedLabel = status === TokenStatus.Removed;
+
+  if (!showNewLabel && !showRemovedLabel) {
+    return null;
+  }
+
+  return (
+    <PillsBase
+      className={clsx('flex-shrink-0', {
+        'bg-negative-100 text-negative-400': showRemovedLabel,
+        'bg-success-100 text-success-400': showNewLabel,
+      })}
```

Comment:
Just a small note: I propose using a `variant` prop instead of passing a class to this component. However, this change would require refactoring the entire `PillsBase` component and its usage. Therefore, it should be handled in a separate maintenance ticket. :)

Reply:
I created issue for that, no changes needed right now 😊 : https://github.com/JoinColony/colonyCDapp/issues/2666